README.md

```diff
@@ -593,6 +593,21 @@ DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket \
   string:"org.freedesktop.timedate1"
 ```
 
+Changing Avahi hostname:
+
+```shell
+DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket \
+  dbus-send \
+  --system \
+  --print-reply \
+  --reply-timeout=2000 \
+  --type=method_call \
+  --dest=org.freedesktop.Avahi \
+  / \
```

Comment:
I guess this line (`/ \` - just a slash) is intentional, but thought I'd ask, just in case.

Reply:
yeah, it's actually a forward-slash that designates some "root" resource in DBUS-land.